### DISCUSSION TITLE
How can I create a PDF file using Quarto with a square paper size, such as 1000 pixels by 1000 pixels?

### DISCUSSION THREAD

**Discussion by shammu119** [quarto-dev/quarto-cli#7244] [Q&A]
### Description

This is my current code. 

``
---
title: "Tourist Arrival"
format:
  pdf: 
    latex:
      geometry:
        - width: "100in"
        - height: "100in"
        - margin: "0.8in"
---
``

**Comment by mcanouil** [comment]
Could you fix your post formatting? See below.

See papersize option at <https://quarto.org/docs/reference/formats/pdf.html#layout>.

Since underlying it's LaTeX, in addition to the documentation page above I suggest to look on the Internet how do this in LaTeX, then use that with the proper options in Quarto.
Also note that your document is invalid, see <https://quarto.org/docs/output-formats/pdf-basics.html#output-options>.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````


---

An example:

````qmd
---
title: "Reproducible Quarto Document"
format:
  pdf:
    geometry:
      - paperwidth=10in
      - paperheight=5in
---

This is a reproducible Quarto document using `format: pdf`.
It is written in Markdown.

The end.
````
